src/Admin.php

```diff
@@ -25,4 +25,89 @@ public static function admin_notices() {
     <?php }
   }
 
+  /**
+   * Returns products backlinking to a given accessory product.
+   */
+  public static function wp_ajax_related_accessories_backlinks() {
+    global $wpdb;
+    $postId = intval($_POST['post']);
+    $html = '<ol>';
+    if($postId)
+    {
+      $params = [
+        'post_type' => ['product', 'product_variant'],
+        'post_status' => 'any',
+        'meta_query' => [
+          ['key' => 'field_group_related_accessories_care_products',
+            'value' => '"' . $postId . '"',
+            'compare' => 'LIKE',
+          ],
+        ],
+        'posts_per_page' => -1,
+      ];
+      $wc_query = new \WP_Query($params);
+      global $post, $product;
+      if ($wc_query->have_posts()) {
+        while ($wc_query->have_posts()) {
+          $wc_query->the_post();
+          $html .= '<li><a href="'.get_edit_post_link($wc_query->post).'">';
+          $html .= esc_html($wc_query->post->post_title);
+          $html .= '</a></li>';
+        }
+        $html .= '</ol>';
+      }
+      else $html = 'No results';
+    }
+    echo $html;
+    wp_die();
+  }
+
+  /**
+   * Related accessories backlinks meta box action.
+   */
+  public static function related_accessories_backlinks_meta_box_setup() {
+    add_action( 'add_meta_boxes', __CLASS__ . '::related_accessories_backlinks_add_meta_box' );
+  }
+
+  /**
+   * Related accessories backlinks meta box configuration.
+   */
+  public static function related_accessories_backlinks_add_meta_box() {
+    add_meta_box(
+      'related-accessories-backlinks',
+      esc_html__( 'Related Accessories', 'woocommerce-related-accessories' ).' backlinks',
```

Comment:
1. The resulting string is not translatable.
2. We trust string translations to be secure.
3. Please use the L10N constant whenever referring to the gettext string translation domain.

```suggestion
      __('Related Accessories Backlinks', Plugin::L10N),
```